extension/src/test/ALParser.test.ts

```diff
@@ -53,15 +53,17 @@ suite("Classes.AL Functions Tests", function () {
       "SharedAccessSignature",
       " ",
       "",
+      "none",
     ];
     const expectedCaptions = [
       "Invoice Posting (v.xx)",
       "Shared access signature (SAS)",
       " ",
       "",
+      "No note",
     ];
     assert.strictEqual(
-      expectedCaptions.length,
+      expectedValues.length,
```

Comment:
@theschitz  Seems as if we both where drunk that weekend... 😂

Reply:
Don't drink (too much) and code :sweat_smile:

Reply:
That's why we have code review, isn't that so?! 😅

Reply:
Can't seem to find an alco lock setting in GitHub :man_shrugging:

Reply:
Exactly!! It's not our fault if we mess up!